source/content/guides/integrated-composer/05-ic-upstreams.md

```diff
@@ -29,47 +29,31 @@ Upstream refers to the source code that is hosted in the Pantheon code repositor
 
 ### How to Add Dependencies to Your Upstream
 
-1. [Clone the Git repository](/guides/git/git-config#clone-your-site-codebase) from the Site Dashboard.
+1. Clone the Git repository for your upstream.
 
-1. Change into the Upstream's configuration directory:
-
-   - Drupal:
-
-    ```bash{promptUser: user}
-    cd upstream-configuration
-    ```
-
-   - WordPress:
```

Comment:
Above the docs refer to `https://github.com/pantheon-upstreams/wordpress-project`. We should decide on a strategy for WordPress, i.e. are we going to copy the upstream management code from drupal-composer-managed into wordpress-project, or are we going to recommend the Bedrock Composer project here? We probably shouldn't publish this page until we decide; if it is going to be a while before we do either, then maybe we could make a note on how to copy this code from drupal-composer-mananaged yourself. Such instructions would be helpful for folks on older upstreams (although maybe we want to just tell them to use the conversion tool to get on the current upstream).

Reply:
This is no longer valid as this is now a composer package that could be pulled anywhere